OpenRA.Mods.Common/Scripting/Properties/AircraftProperties.cs

```diff
@@ -8,7 +8,7 @@
  * information, see COPYING.
  */
 #endregion
-
+ 
```

Comment:
OpenRA.Mods.Common/Scripting/Properties/AircraftProperties.cs:L11: [SP2000] Invalid spacing at the end of the line.